specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/network/rfc7540/connection.management/server.sent.read.abort.before.correlated/client.rpt

```diff
@@ -48,14 +48,16 @@ read [0x00 0x00 0x00]                   # length = 0
      [0x01]                             # ACK
      [0x00 0x00 0x00 0x00]              # stream_id = 0
 
-write [0x00 0x00 0x13]                  # length = 19
-      [0x01]                            # HEADERS frame
-      [0x04]                            # END_HEADERS
-      [0x00 0x00 0x00 0x01]             # stream_id = 1
-      [0x82]                            # :method: GET
-      [0x86]                            # :scheme: http
-      [0x84]                            # :path: /
-      [0x01] [0x0e] "localhost:8080"    # :authority: localhost:8080
+write [0x00 0x00 0x33]                                      # length = 51
```

Comment:
I think we need to revisit the `correlated` naming concept here to properly capture the intent.